README.md

```diff
@@ -1,735 +1,1009 @@
 
-> 💬 The Lending contract accepts ETH deposits and allows depositor's to take out a loan in CORN 🌽. The contract tracks each depositor's address and only allows them to borrow as long as they maintain at least 120% of the loans value in ETH. If the collateral falls in value or if CORN goes up in value then the borrower's position may be liquidatable by anyone who pays back the loan. The liquidator has an incentive to do this because they collect a 10% fee on top of the value of the loan. 
+🤔 How do they maintain their peg? There are several mechanisms:
 
-> 📈 The Lending contract naively uses the price returned by a CORN/ETH DEX contract. This makes it easy for you to change the price of CORN by "moving the market" with large swaps. Shout out to the [DEX challenge](https://github.com/scaffold-eth/se-2-challenges/blob/challenge-4-dex/README.md)! Using a DEX as the sole price oracle would never work in a production grade system but it will help to demonstrate the different market conditions that affect a lending protocol.
+- 💎 **Collateralization**: Users lock up valuable assets (like ETH) as collateral to mint stablecoins. This ensures each stablecoin is backed by real value.
+- 📊 **Interest Rates**: By adjusting borrowing and savings rates, we can influence supply and demand to maintain the peg.
+- 🚨 **Liquidations**: If collateral value drops too low, positions can be liquidated to protect the system.
+- 💸 **Market Operations**: The system can incentivize buying or selling to maintain the peg.
 
-> 🌽 Your job is to fill out the functions in the Lending contract so that it enables you to take out a CORN loan.
+👍 Now that you understand the basics, let's build our own stablecoin system!
 
-> 💬 Meet other builders working on this challenge and get help in the TODO
+</details>
+
+🔍 First we should mention there are lots of different types of stablecoins on the market. Some are backed 1-1 with actual USD denominated assets in a bank (USDC, USDT). Others are backed by crypto and use special mechanisms to maintain their peg (Dai, RAI, LUSD/BOLD).
+
+📚 This challenge is modeled after one of the first crypto-backed stablecoins called Dai - back when the only thing backing it was ETH. Later Dai would allow multiple types of collateral and change it's design somewhat so the version we are building is commonly referred to as "single collateral Dai".
+
+⚠️ You are highly encouraged to have completed the [Over-collateralized Lending challenge](https://speedrunethereum.com/challenge/over-collateralized-lending) prior to attempting this one since we will be building on that same basic system but won't be discussing it in detail.
 
 ---
 
 ## Checkpoint 0: 📦 Environment 📚
 
-Before you begin, you need to install the following tools:
+🛠️ Before you begin, you need to install the following tools:
 
 - [Node (v18 LTS)](https://nodejs.org/en/download/)
```

Comment:
Should we change this to 
```suggestion
- [Node (>=v18)](https://nodejs.org/en/download/)
```
now that v18 has [reached it's end of life](https://nodejs.org/en/about/previous-releases)?